### PR TITLE
Generate bookable slots in a background job

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -251,6 +253,10 @@ GEM
       sprockets (>= 2.8, < 4.0)
       sprockets-rails (>= 2.0, < 4.0)
       tilt (>= 1.1, < 3)
+    sidekiq (4.1.1)
```
Comment (benlovell) [inline Gemfile.lock]: Can you update sidekiq to `4.2.2`?

Comment (AndrewVos) [inline Gemfile.lock]: done

**Comment by AndrewVos** [comment]
@benlovell Done all that

**Comment by benlovell** [comment]
Loveliness 💯

**Pull request merged**